### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# sedy_test
\ No newline at end of file
+# sedy_test
+
+Attention à la consistence
```
Comment (djhi) [inline README.md]: s/consistence/cohérence

Comment (djhi) [inline README.md]: s/consistence/cohérence

Comment (djhi) [inline README.md]: s/consistence/cohérence

Comment (djhi) [inline README.md]: s/consistence/cohérence

Comment (djhi) [inline README.md]: s/consistence/cohérence

Comment (djhi) [inline README.md]: s/consistence/cohérence

Comment (djhi) [inline README.md]: s/consistence/cohérence

Comment (djhi) [inline README.md]: s/consistence/cohérence

Comment (djhi) [inline README.md]: s/consistence/cohérence

Comment (djhi) [inline README.md]: s/consistence/cohérence

Comment (djhi) [inline README.md]: s/consistence/cohérence

Comment (djhi) [inline README.md]: s/consistence/cohérence

Comment (djhi) [inline README.md]: s/consistence/cohérence/

Comment (djhi) [reply README.md]: s/consistence/cohérence/

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# sedy_test
\ No newline at end of file
+# sedy_test
```
Comment (Kmaschta) [inline README.md]: s/test/prod/

Comment (Kmaschta) [inline README.md]: s/test/prod/

Comment (Kmaschta) [inline README.md]: s/test/prod/

Comment (Kmaschta) [inline README.md]: s/test/prod/

Comment (Kmaschta) [inline README.md]: s/test/prod/

**Pull request merged**